core/primitives-core/src/version.rs

```diff
@@ -445,7 +450,7 @@ pub const PROD_GENESIS_PROTOCOL_VERSION: ProtocolVersion = 29;
 pub const MIN_SUPPORTED_PROTOCOL_VERSION: ProtocolVersion = 77;
 
 /// Current protocol version used on the mainnet with all stable features.
-const STABLE_PROTOCOL_VERSION: ProtocolVersion = 80;
+const STABLE_PROTOCOL_VERSION: ProtocolVersion = 81;
```

Comment:
80 is set to go out as 2.8 so 81 can be the next protocol upgrade with this change included.